gnome-initial-setup/pages/account/gis-account-page.c

```diff
@@ -161,13 +163,52 @@ switch_login_mode (GisAccountPage *page)
     set_mode (page, UM_LOCAL);
 }
 
+static void
+enable_endlessm_metrics (void)
```

Comment:
Can we put this function in a common location so that it can be shared between the accounts page and here, rather than essentially copy-pasted (apart from the GtkToggle binding)?

Reply:
The place would be `gis-page-util.[ch]` which is full of our downstream functionality that is used in >1 page